app/src/main/res/layout/activity_parking_stay.xml

```diff
@@ -0,0 +1,169 @@
+        android:text="@string/start_time_selected_text_view"
+        app:layout_constraintBottom_toTopOf="@id/end_date_selected_text_view"
+        app:layout_constraintStart_toEndOf="@id/start_date_text_view"
+        app:layout_constraintEnd_toStartOf="@id/start_date_time_button"
+        app:layout_constraintTop_toBottomOf="@id/start_date_selected_text_view" />
+
+    <TextView
+        style="@style/text_view_style_parking_stay_activity"
+        android:id="@+id/end_date_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/date_end_text"
+        app:layout_constraintBottom_toTopOf="@id/parking_lot_text_view"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/end_time_selected_text_view"
+        app:layout_constraintTop_toBottomOf="@id/start_date_text_view" />
+
+    <TextView
+        style="@style/text_view_selected_style_parking_stay_activity"
+        android:id="@+id/end_date_selected_text_view"
+        android:layout_width="@dimen/text_view_width_parking_stay_activity"
+        android:layout_height="wrap_content"
+        android:text="@string/end_date_selected_text_view"
+        app:layout_constraintBottom_toTopOf="@id/end_time_selected_text_view"
+        app:layout_constraintStart_toEndOf="@id/end_date_text_view"
+        app:layout_constraintEnd_toStartOf="@id/end_date_time_button"
+        app:layout_constraintTop_toBottomOf="@id/start_time_selected_text_view" />
+
+    <TextView
+        style="@style/text_view_selected_style_parking_stay_activity"
+        android:id="@+id/end_time_selected_text_view"
+        android:layout_width="@dimen/text_view_width_parking_stay_activity"
+        android:layout_height="wrap_content"
+        android:text="@string/end_time_selected_text_view"
+        app:layout_constraintBottom_toTopOf="@id/parking_lot_edit_text"
+        app:layout_constraintStart_toEndOf="@id/end_date_text_view"
+        app:layout_constraintEnd_toStartOf="@id/end_date_time_button"
+        app:layout_constraintTop_toBottomOf="@id/end_date_selected_text_view" />
+
+    <TextView
+        style="@style/text_view_style_parking_stay_activity"
+        android:id="@+id/parking_lot_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/parking_lot_text"
+        app:layout_constraintBottom_toTopOf="@id/security_code_text_view"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/parking_lot_edit_text"
+        app:layout_constraintTop_toBottomOf="@id/end_date_text_view" />
+
+    <EditText
+        style="@style/edit_text_style_parking_stay_activity"
+        android:id="@+id/parking_lot_edit_text"
+        android:layout_width="@dimen/edit_text_width_parking_stay_activity"
+        android:layout_height="wrap_content"
+        android:autofillHints=""
```

Comment:
is this really necessary? if so, what is this for?

Reply:
not really necessary. Deleted